### PR TITLE
added accept ranges support required by safari to play videos

### PR DESCRIPTION
**Pull request by kevinduong**

### CODE REVIEW

File: src/UmbracoFileSystemProviders.Azure/FileSystemVirtualFile.cs [hunk 1]
```diff
@@ -65,6 +65,9 @@ public override Stream Open()
                 cache.SetCacheability(HttpCacheability.Public);
                 cache.VaryByHeaders["Accept-Encoding"] = true;
 
+                // Add Accept-Ranges header to fix videos not playing on Safari
```
Comment (JimBobSquarePants) [inline src/UmbracoFileSystemProviders.Azure/FileSystemVirtualFile.cs]: Is it really that easy?! If, so that's amazing! 😄

**Comment by JimBobSquarePants** [comment]
From what information I can find this should cover our needs with this provider. @Jeavon Do you concur?

File: src/UmbracoFileSystemProviders.Azure/FileSystemVirtualFile.cs (hunk 1 above)
Comment (kevinduong) [inline src/UmbracoFileSystemProviders.Azure/FileSystemVirtualFile.cs]: Do we want to make it so that it only applies this header for mp4 files?

**Comment by Jeavon** [comment]
@JimBobSquarePants looks brilliantly simple, lets merge and try out the pre-release?

File: src/UmbracoFileSystemProviders.Azure/FileSystemVirtualFile.cs (hunk 1 above)
Comment (JimBobSquarePants) [inline src/UmbracoFileSystemProviders.Azure/FileSystemVirtualFile.cs]: No we'll add it for all. Should be fine.

**Comment by JimBobSquarePants** [comment]
There we go! All merged in.

**Pull request merged**